src/website/app/demos/Grid/examples/Grid/gutters.js

```diff
@@ -0,0 +1,36 @@
+/* @flow */
+import GridItem from '../../components/GridItem';
+import DemoLayout from '../../components/DemoLayout';
+import Grid from '../../components/Grid';
+
+export default {
+  id: 'gutters',
+  title: 'Gutters',
+  description: `Grid will add a default horizontal gap between items. Use the
+\`gutterWidth\` prop to define a different value, including \`0\` to disable
+gutters altogether.
+
+_Numbers will be appended with \`px\`. String values will be passed directly._`,
+  scope: { DemoLayout, Grid, GridItem },
+  source: `
+    () => {
+      const propValues = [
+        'md', // default
+        'xl',
+        '1.5em',
+        50,
+        0
+      ];
+
+      const renderGrides = () =>
+        propValues.map((value, index) => (
+          <Grid gutterWidth={value} key={index}>
+            <GridItem>A</GridItem>
+            <GridItem>B</GridItem>
+            <GridItem>C</GridItem>
+          </Grid>
+        ));
+
+      return <DemoLayout>{renderGrides()}</DemoLayout>;
+    }`
+};
```

Comment:
This example might be more visually appealing if the rendered gutter widths were ordered from least to greatest or greatest to least.

Reply:
Also updated Flex's gutter example.